src/renderer/src/app/App.tsx

```
@@ -0,0 +1,12 @@
+import Home from '@renderer/pages/Home'
+import { Route, Routes } from 'react-router-dom'
+
+const App = () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)